setup.py

```diff
@@ -103,10 +129,53 @@ def get_version():
             'Programming Language :: Python :: 3.6',
         ],
         license='Apache License 2.0',
-        setup_requires=['pytest-runner'],
+        setup_requires=['pytest-runner', 'cython', 'torch>=1.1', 'numpy'],
         tests_require=['pytest'],
         install_requires=[
             'mmcv>=0.2.6', 'numpy', 'matplotlib', 'six', 'terminaltables',
-            'pycocotools'
+            'pycocotools', 'torch>=1.1'
+        ],
+        ext_modules=[
+            make_cython_ext(
+                name='soft_nms_cpu',
+                modpath=('mmdet', 'ops', 'nms'),
+                sources=['src/soft_nms_cpu.pyx']),
+            make_cuda_ext(
+                name='roi_align_cuda',
+                modpath=('mmdet', 'ops', 'roi_align'),
+                sources=['src/roi_align_cuda.cpp', 'src/roi_align_kernel.cu']),
+            make_cuda_ext(
+                name='roi_pool_cuda',
+                modpath=('mmdet', 'ops', 'roi_pool'),
+                sources=['src/roi_pool_cuda.cpp', 'src/roi_pool_kernel.cu']),
+            make_cuda_ext(
+                name='nms_cpu',
+                modpath=('mmdet', 'ops', 'nms'),
+                sources=['src/nms_cpu.cpp']),
+            make_cuda_ext(
+                name='nms_cuda',
+                modpath=('mmdet', 'ops', 'nms'),
+                sources=['src/nms_cuda.cpp', 'src/nms_kernel.cu']),
+            make_cuda_ext(
+                name='deform_conv_cuda',
+                modpath=('mmdet', 'ops', 'dcn'),
+                sources=['src/deform_conv_cuda.cpp',
+                         'src/deform_conv_cuda_kernel.cu']),
+            make_cuda_ext(
+                name='deform_pool_cuda',
+                modpath=('mmdet', 'ops', 'dcn'),
+                sources=['src/deform_pool_cuda.cpp',
+                         'src/deform_pool_cuda_kernel.cu']),
+            make_cuda_ext(
+                name='sigmoid_focal_loss_cuda',
+                modpath=('mmdet', 'ops', 'sigmoid_focal_loss'),
+                sources=['src/sigmoid_focal_loss.cpp',
+                         'src/sigmoid_focal_loss_cuda.cu']),
+            make_cuda_ext(
+                name='masked_conv2d_cuda',
+                modpath=('mmdet', 'ops', 'masked_conv'),
+                sources=['src/masked_conv2d_cuda.cpp',
+                         'src/masked_conv2d_kernel.cu']),
```

Comment:
It seems that the code was not formatted by `yapf` with the specified [configuration](https://github.com/open-mmlab/mmdetection/blob/master/.style.yapf).

Reply:
Will do